package.json

```diff
@@ -31,17 +31,21 @@
     "del": "^2.2.0",
     "ejs": "^2.5.1",
     "es5-shim": "^4.5.2",
+    "eslint-config-standard": "^10.2.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-node": "^4.2.2",
+    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-standard": "^3.0.1",
     "faker": "^3.1.0",
     "fs.extra": "^1.3.2",
     "gulp": "^3.8.7",
     "gulp-babel": "^6.1.2",
-    "gulp-clean": "^0.3.1",
+    "gulp-clean": "^0.3.2",
```

Comment:
If we're updating dependencies, the `yarn.lock` file should probably be updated as well.